content/competitions/codejam.html

```diff
@@ -0,0 +1,107 @@
+                efficient and reliable. Points are awarded for the number of tests you pass and the question’s
+                difficulty.</p>
+            <p>There will be a leaderboard of competitors based on points scored. The top scorers will get prizes!</p>
+
+
+            <h3>How can I get ready?</h3>
+            <p>You will need a HackerRank account to compete, <a href="https://www.hackerrank.com/">sign up here</a>.
+                You don’t need to do special practice or
+                preparation, just come well-rested and ready to think!</p>
+            <p>To get familiar with the HackerRank interface and styles of questions, you can try some practice
+                questions on HackerRank.
+            </p>
+
+            <h3>What do I do on the day?</h3>
+            <p>The CodeJam will run from 10am – 3:30pm on Saturday 28th March. You can join in anytime but the more time
+                you spend, the better your chances of winning prizes! </p>
+            <p>We’ll post updates and reminders throughout the day on our <a href="http://slack.uqcs.org/">Slack
+                    chat</a>.
+                Presentations and prizes will be live streamed on our <a
+                    href="https://www.youtube.com/c/UQComputingSociety">YouTube</a>, at the start and end of the day.
+                The actual coding competition will be hosted on <a href="https://www.hackerrank.com/">HackerRank</a>.
+            </p>
+            <p>If you are coming in person, make sure to get a ticket <a href="https://www.eventbrite.com.au/e/uqcs-codejam-presented-by-imc-tickets-144518525801">here</a> 
+                for COVID-19 contact tracing purposes!
+            </p>
+
+            <h3>Prizes!</h3>
+            <p>We have various prizes, all sponsored by <a href="https://www.imc.com/ap/">IMC</a>.</p>
+            <table>
+
+                <tbody>
+                    <tr>
+                        <th>1st place overall</th>
+                        <td>$150 JB Hi-Fi gift voucher</td>
+                    </tr>
+                    <tr>
+                        <th>2nd place overall</th>
+                        <td>$100 JB Hi-Fi gift voucher</td>
+                    </tr>
+                    <tr>
+                        <th>3rd place overall</th>
+                        <td>$60 JB Hi-Fi gift voucher</td>
+                    </tr>
+                    <tr>
+                        <th>Top first-year</th>
+                        <td>$120 JB Hi-Fi gift voucher</td>
+                    </tr>
+                </tbody>
+            </table>
+            <small>
+                <p>
+                    Limit of one prize per person. 
+                    To be eligible for prizes, you must be a current university student and registered on the HackerRank contest.
+                </p>
+                <p>
+                    You are considered a <i>first-year</i> if you have not completed CSSE2310,
```

Comment:
We did plan to change the first-year requirements to be simpler with fewer edge cases (@JamesDearlove).